src/EditorFeatures/CSharpTest/MoveDeclarationNearReference/MoveDeclarationNearReferenceTests.cs

```diff
@@ -1461,5 +1821,23 @@ void M()
     }
 }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMoveDeclarationNearReference)]
+        public async Task TestUsedInMultipleSwitchSections_TopLevelStatement_CannotMove()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"
+int [||]x;
+switch (true)
+{
+    case true:
+        x = 0;
+        break;
+    case false:
+        x = 0;
+        break;
+}
+");
+        }
```

Comment:
could we have one more test with a local outside of a local function?  thanks!

Reply:
Which current test would that mirror?

Reply:
don't know :)  just something i want to make sure works given that it croses scopes, and each of these would be in a global statement.  i just want to make sure that doesn't break :)

Reply:
Test added